src/common/UserAvatar.js

```diff
@@ -45,24 +55,25 @@ function UserAvatar(props: Props) {
   }
 
   return (
-    <View>
-      <Touchable onPress={onPress} style={style}>
-        <ImageBackground
```

Comment:
The commit message says

    This will allow us to use an `Icon` in place of a avatar URL in the
    future. The implementation of `ImageBackground` is very simple, we
    essentially inline it.

But when I look at the implementation in `node_modules/react-native/Libraries/Image/ImageBackground.js`, it seems like there are some subtleties there that aren't carried through in this version.

* The upstream implementation has an `accessibilityIgnoresInvertColors={true}`. I'm not sure what that does; is it not needed for our case?
* The upstream implementation provides the following style props:
```js
{
  position: 'absolute',
  left: 0,
  right: 0,
  top: 0,
  bottom: 0,
}
```
(under the name of `StyleSheet.absoluteFill`). What's the effect of those?

Reply:
Re: `accessibilityIgnoresInvertColors` ­— this apparently overrides the user's "Invert Colors" setting for just this view (see the [Apple docs](https://developer.apple.com/documentation/uikit/uiview/2865843-accessibilityignoresinvertcolors)). I guess it makes sense to keep this, I'll do that in a followup commit.

I found that setting the `width` and `height` was sufficient to cause the avatar to take up the required space, so I omitted the options in `absoluteFill` to keep things simple. It's possible there is a circumstance that causes them to be necessary, but if so, I don't think it appears in our app.